CMakeLists.txt

```diff
@@ -16,7 +16,8 @@ if (BUILD_DEPS)
     set(AWS_DEPS_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR}/deps)
     set(AWS_DEPS_INSTALL_DIR ${CMAKE_CURRENT_BINARY_DIR}/deps/install)
 
-    list(APPEND CMAKE_MODULE_PATH "${AWS_DEPS_INSTALL_DIR}/${CMAKE_INSTALL_LIBDIR}/cmake")
+    list(APPEND CMAKE_MODULE_PATH "${AWS_DEPS_INSTALL_DIR}/lib/cmake")
+    list(APPEND CMAKE_MODULE_PATH "${AWS_DEPS_INSTALL_DIR}/lib64/cmake")
```

Comment:
If you do my trick with the gnuinstall dirs stuff, then use CMAKE_INSTALL_LIBDIR you can avoid this.

Also probably want to use CMAKE_PREFIX_PATH

Reply:
I tried that, it did not work, I always got lib64 here, and all the other libs got lib, despite using GNUInstallDirs. I didn't troubleshoot it further yet.